packages/freshchat-api/src/index.ts

```diff
@@ -455,15 +455,13 @@ export default class Freshchat {
    */
   conversationPropertiesUpdate(
     conversationId: string,
-    status: string,
     properties: unknown,
-    assigned_agent_id: string,
+    status?: string | null,
   ): AxiosPromise<Conversation> {
     const conversationPropsUpdateApiUrl = `${this.apiUrl}/conversations/${conversationId}`;
-    let requiredProperties = JSON.stringify({
-        properties,
-        status,
-      });
+    
+    const payload = status === null ? { properties } : { properties, status };
```

Comment:
Please optimize the code 

`const payload = { properties, ...(status !== null && { status })};
`

Reply:
it reduces readability in my opinion